### ISSUE TITLE
Formatting of `@> array[]` should work again

### ISSUE THREAD

**Issue by StephanHoyer** [sql-formatter-org/sql-formatter#275]
**Input data**

Formatting fails in 8.x, worked in 7.x

```sql
-- ADD SQL HERE
`select * where "meta_values" @> array['6d74db4a-61bf-410d-8c5d-6919b5ce50a4'::uuid]`
```

**Expected Output**

```sql
-- ADD SQL HERE
select
  *
where
  "meta_values" @ > array ['6d74db4a-61bf-410d-8c5d-6919b5ce50a4':: uuid]
```

**Actual Output**

Error
```
TypeError: Cannot read properties of undefined (reading 'children')
    at InlineBlock.inlineWidth (.../node_modules/sql-formatter/lib/core/InlineBlock.js:55:62)
    at InlineBlock.isInlineBlock (.../node_modules/sql-formatter/lib/core/InlineBlock.js:48:19)
    at ExpressionFormatter.formatParenthesis (.../node_modules/sql-formatter/lib/core/ExpressionFormatter.js:134:37)
    at ExpressionFormatter.formatArraySubscript (.../node_modules/sql-formatter/lib/core/ExpressionFormatter.js:129:12)
    at ExpressionFormatter.format (.../node_modules/sql-formatter/lib/core/ExpressionFormatter.js:84:18)
    at ExpressionFormatter.formatSubExpression (.../node_modules/sql-formatter/lib/core/ExpressionFormatter.js:215:10)
    at ExpressionFormatter.formatClause (.../node_modules/sql-formatter/lib/core/ExpressionFormatter.js:176:26)
    at ExpressionFormatter.format (.../node_modules/sql-formatter/lib/core/ExpressionFormatter.js:96:18)
    at SqlFormatter.formatStatement (.../node_modules/sql-formatter/lib/core/Formatter.js:110:10)
    at .../node_modules/sql-formatter/lib/core/Formatter.js:100:22
    at Array.map (<anonymous>)
    at SqlFormatter.formatAst (.../node_modules/sql-formatter/lib/core/Formatter.js:99:25)
    at SqlFormatter.format (.../node_modules/sql-formatter/lib/core/Formatter.js:90:33)
    at format (.../node_modules/sql-formatter/lib/sqlFormatter.js:120:33)
```

**Usage**

- How are you calling / using the library?
```javascript
   format(
    `select * where "meta_values" @> array['6d74db4a-61bf-410d-8c5d-6919b5ce50a4'::uuid]`
  )´
```

- What SQL language(s) does this apply to?
postgres

- Which SQL Formatter version are you using?
8.0

**Comment by nene** [comment]
Which SQL dialect is this? Postgres?

**Comment by StephanHoyer** [comment]
BTW: Great lib!

**Comment by nene** [comment]
Looking into this, the problems is not really in the `@>` operator at all. Instead it's the array literal syntax `array[]` which breaks things.

Somehow the fact that Postgres also supports arrays was overlooked during the refactoring of square brackets handling in 8.0.

**Comment by nene** [comment]
Released a fix in 8.0.1

Thanks for reporting!

**Comment by StephanHoyer** [comment]
great, works like a charm. thank you!
